### DISCUSSION TITLE
Assistants example

### DISCUSSION THREAD

**Discussion by Strange-Knoll** [64bit/async-openai#145] [Q&A] [answered]
I noticed the request for an assistants example in the examples README. I have made one but im new to using github collaboratively and am not sure how to properly share this example linked below.
https://github.com/Strange-Knoll/async-openai-assistants-example

**Comment by 64bit** [comment, marked as answer]
Thank you for offering to contribute!

For colloborating on Github, the workflow is based on [pull requests](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests). More specifically [pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)

So essentially your steps would look like:
1. fork this repo into your personal account
2. create a branch and push changes to your copy of this repo in personal account
3. When you create the pull request - you'll select upstream repo to be this one and `main` branch. 

-----

Regarding the example - I'd recommend creating it inside `examples/<your-example-name>` directory along with companion README for users to know what it does.
